src/main.cpp

```diff
@@ -30,6 +30,19 @@ using namespace aos::zephyr;
 
 int main(void)
 {
+    atexit([]() {
```

Comment:
Is it compiled for target platforms?
If so, move it before apps.Init

Reply:
Successfully compiled for rpi5 board